versions/3.0.3.md

```diff
@@ -317,15 +317,15 @@ url: https://www.apache.org/licenses/LICENSE-2.0.html
 
 #### <a name="serverObject"></a>Server Object
 
-An object representing a Server.
+An object representing a single server or a group of servers sharing a common URL structure.
 
 ##### Fixed Fields
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="serverUrl"></a>url | `string` | **REQUIRED**. A URL to the target host.  This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served. Variable substitutions will be made when a variable is named in `{`brackets`}`.
+<a name="serverUrl"></a>url | `string` | **REQUIRED**. A URL to the target host.  This URL may include variable expressions, delimited by `{`curly braces`}`.  Each variable expression MUST correspond to a [Server Variable Object](#serverVariableObject) named in the `variables` field, and the API consumer MUST replace each variable expression with a with a valid value, as specified by the corresponding Server Variable Object.  The URL MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served.
```

Comment:
We should not talk about how to resolve relative URLs here. We have a dedicated paragraph for that.  We could link to it.